### ISSUE TITLE
need to load pretrained models？

### ISSUE THREAD

**Issue by ZhangLei999** [jinxinzhou/DREAM#4]
Great work! I have questions about the following line of code:

noise_pred = self.denoise_fn(torch.cat([x_in['SR'], x_noisy], dim=1), continuous_sqrt_alpha_cumprod)
noise_rec = self.denoise_fn(torch.cat([x_in['SR'], x_noisy_pred], dim=1), continuous_sqrt_alpha_cumprod)
loss_func(noise_pred, noise_rec)

Do we need to first train the network using traditional MSE loss(noise_pred, noise)?
If not trained, the values of noise_pred and noise_rec are meaningless.
